src/Controllers/photos.controllers.ts

```diff
@@ -0,0 +1,53 @@
+import axios from 'axios';
+import {JSONPhoto} from '../Models/interfaces';
+import {check_valid_rover} from '../Middleware/check_valid_rover.middleware';
+import {API_BASE, API_KEY} from '../Routes/server'
+
+export async function photos(req: any, res: any) {
```

Comment:
this function is a little long. Some suggestions:
- move validation to a separate file
- fetching data: use a service